### PR TITLE
[DEV-3421] Instant access backend

### PR DESCRIPTION
**Pull request by slisaasquatch**
## Description of the change

Schema changes for instant access widgets

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation or Development tools (readme, specs, tests, code formatting)

## Links

- Jira issue number: DEV-3421
- Process.st launch checklist: https://app.process.st/runs/Instant-Access-Widgets-Backend-lZjkdKLnZgpvdxkpJEFDtw/tasks/ipJPSwLUytFe3xMt6P5PYw

### Development

- [ ] [Prettier](https://www.npmjs.com/package/prettier) was run (if applicable)
- [ ] The behaviour changes in the pull request are covered by specs
- [ ] All tests related to the changed code pass in development

### Paperwork

- [ ] This pull request has a descriptive title and information useful to a reviewer
- [ ] This pull request has a Jira number
- [ ] This pull request has a Process.st launch checklist

### Code review

- [ ] Changes have been reviewed by at least one other engineer
- [ ] Security impacts of this change have been considered

### CODE REVIEW

**Review by johanventer — COMMENTED** [review]
Looks good, needs package publish before final approval

File: package.json
Comment (johanventer) [inline package.json:3]: Why has the version of the schema gone backwards?

**Review by johanventer — CHANGES_REQUESTED** [review]

File: package.json
Comment (slisaasquatch) [reply package.json:3]: Because this branch is older than the latest version on production

**Review by johanventer — DISMISSED** [review]

**Review by DerekSiemens — APPROVED** [review]

**Pull request merged**
